platform/darwin/src/MGLStyle.h

```diff
@@ -162,22 +164,46 @@ static const NSInteger MGLStyleDefaultVersion = 9;
  */
 + (NSURL *)satelliteStreetsStyleURLWithVersion:(NSInteger)version;
 
-//- (instancetype)init NS_UNAVAILABLE;
-
-#pragma mark - Runtime style API
+#pragma mark Runtime Styling
 
-- (id <MGLStyleLayer>)layerWithIdentifier:(NSString *)identifier;
+/**
+ Returns a layer that conforms to MGLStyleLayer if any layer with the given identifier was found.
+ 
+ @return layer A layer of type `MGLBackgroundStyleLayer`, `MGLCircleStyleLayer`,
+ `MGLFillStyleLayer`, `MGLLineStyleLayer`, `MGLRasterStyleLayer` or `MGLSymbolStyleLayer`.
```

Comment:
If we wanted to include a list, we could use `@see`. Something like this:

```
 @see `MGLStyle`, `MGLMapView.styleURL`, `delegate`
```

Ends up looking like this in jazzy:

![screen shot 2016-08-10 at 3 05 51 pm](https://cloud.githubusercontent.com/assets/1198851/17567137/257551ca-5f0c-11e6-8f20-dabe47021da7.png)